examples/ruby/spec/bidi/logging_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper'
 
-RSpec.describe 'Logging' do
+RSpec.describe 'Logging', exclusive: {bidi: true, reason: 'only executed when bidi is enabled'},
```

Comment:
These are the guards registered in this test suite:
https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples/ruby/spec/spec_helper.rb#L19-L22

Either way, we're explicitly starting a bidi session below, so it won't ever run without being enabled.

Finally, guards don't work on the `describe` block, unfortunately, anyway